compiler/rustc_codegen_cranelift/src/abi/mod.rs

```diff
@@ -355,19 +356,19 @@ struct CallArgument<'tcx> {
 // FIXME avoid intermediate `CValue` before calling `adjust_arg_for_abi`
 fn codegen_call_argument_operand<'tcx>(
     fx: &mut FunctionCx<'_, '_, 'tcx>,
-    operand: &Operand<'tcx>,
+    operand: &Spanned<Operand<'tcx>>,
 ) -> CallArgument<'tcx> {
     CallArgument {
-        value: codegen_operand(fx, operand),
-        is_owned: matches!(operand, Operand::Move(_)),
+        value: codegen_operand(fx, &operand.node),
+        is_owned: matches!(operand.node, Operand::Move(_)),
```

Comment:
Maybe use `operand.node` on the call sites of `codegen_call_argument_operand()` instead?

Reply:
Sure, why not. Done!